example/src/release_tests/combo/index.tsx

```diff
@@ -174,41 +176,14 @@ class Combo extends Component<ComboProps> {
               <RectButton
                 style={styles.rectButton}
                 onPress={() => Alert.alert('First row clicked')}>
-                <Text style={styles.buttonText}>
-                  Swipe this row & observe highlight delay
-                </Text>
-                {/* Info icon will cancel when you scroll in the direction of the scrollview
-                  but if you move finger horizontally it would allow you to "re-enter" into
-                  an active state. This is typical for most of the buttons on iOS (but not
-                  on Android where the touch cancels as soon as you leave the area of the
-                  button). */}
-                <InfoButton name="first" />
+                <Button title="first" />
               </RectButton>
             </Swipeable>
             <View style={styles.buttonDelimiter} />
             <RectButton
               style={styles.rectButton}
               onPress={() => Alert.alert('Second row clicked')}>
-              <Text style={styles.buttonText}>
-                Second info icon will block scrolling
-              </Text>
-              {/* Info icon will block interaction with other gesture handlers including
-                  the scrollview handler its a descendant of. This is typical for buttons
-                  embedded in a scrollable content on iOS. */}
-              <InfoButton disallowInterruption name="second" />
-            </RectButton>
-            <View style={styles.buttonDelimiter} />
-            <RectButton
-              style={styles.rectButton}
-              onPress={() => Alert.alert('Third row clicked')}>
-              <Text style={styles.buttonText}>
-                This one will cancel when you drag outside
-              </Text>
-              {/* Info icon will cancel when you drag your finger outside of its bounds and
-                  then back unlike all the previous icons that would activate when you re-enter
-                  their activation area. This is a typical bahaviour for android but less frequent
-                  for most of the iOS native apps. */}
-              <InfoButton shouldCancelWhenOutside name="third" />
```

Comment:
Would it be possible to reimplement this using the new components? I'd like to keep something along these lines to make sure this behavior doesn't change accidentally.

Reply:
done in [0a87e5e](https://github.com/software-mansion/react-native-gesture-handler/pull/3230/commits/0a87e5e1986b3182865548bf903bf00ea52e3f21)